tests/python/common/test_subgraph.py

```diff
@@ -95,6 +95,48 @@ def test_subgraph():
     assert F.allclose(h, g.ndata["h"])
 
 
+def test_subgraph_not_relabel_nodes():
+    g = generate_graph()
+    h = g.ndata["h"]
+    l = g.edata["l"]
+    nid = [0, 2, 3, 6, 7, 9]
+    sg = g.subgraph(nid, relabel_nodes=False)
+    eid = {2, 3, 4, 5, 10, 11, 12, 13, 16}
+    assert set(F.asnumpy(sg.edata[dgl.EID])) == eid
+    eid = sg.edata[dgl.EID]
+    # the subgraph is empty initially except for NID/EID field
+    assert len(sg.edata) == 2
+    sh = sg.ndata["h"]
+    assert F.allclose(
+        F.gather_row(h, F.tensor([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])), sh
+    )
```

Comment:
This is the only difference between test_subgraph_not_relabel_nodes and test_subgraph_relabel_nodes. What does this one do? Add comment?

Reply:
The node number is not reduced. So the subgraph keep the same node information as the original graph.